distribution/test/itests/test-itests-common/src/main/java/org/codice/ddf/itests/common/XSearch.java

```diff
@@ -0,0 +1,77 @@
+/**
+ * Copyright (c) Codice Foundation
+ * <p>
+ * This is free software: you can redistribute it and/or modify it under the terms of the GNU Lesser
+ * General Public License as published by the Free Software Foundation, either version 3 of the
+ * License, or any later version.
+ * <p>
+ * This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without
+ * even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+ * Lesser General Public License for more details. A copy of the GNU Lesser General Public License
+ * is distributed along with this program and can be found at
+ * <http://www.gnu.org/licenses/lgpl.html>.
+ */
+package org.codice.ddf.itests.common;
+
+import javax.xml.xpath.XPath;
+import javax.xml.xpath.XPathExpression;
+import javax.xml.xpath.XPathFactory;
+
+/**
+ * Simpler helper class for XPath searches. Use with the XDocument helper class.
```

Comment:
Need to mention the other classes? Of course not. ;-)

I hate creating extra classes. Before I did so, I explored existing XML-related classes and made experiments with them. My hope was that I could leverage existing classes to do the work. In end, none of the other classes served the needs of the itests very well. However, the XMLUnit library, the hasXPath matcher, and the DDF's own XPathHelper all have good use cases. I wanted to provide a roadmap to others about the strengths of these other XML-related classes. The class comment seemed like the best place to put the guidance. 

I see three options now:
(1) Delete the guidance entirely. This is the best option if the information is not useful to others. The guidance is essentially, a distilled version of my research notes.
(2) Leave the guidance in the class comment.
(3) Move the guidance to another place like a README.md file or wiki page.